tests/__tests__/sitka_module.test.ts

```diff
@@ -67,21 +67,61 @@ describe("SitkaModule", () => {
   })
 
   test('able to get defaultState', () => {
+    const expected = { 
```

Comment:
there are several times that this expected default state is used.  go ahead and create a defaultTextModuleState const at the top of the file that can be used by all tests.